.coderabbit/config.yaml

```
@@ -0,0 +1,8 @@
+autolabel: true
+automerge: false
+review:
+  enabled: true
+  reviewers:
+    - coderabbit
+  instructions: |
+    Please focus on code quality, security, and documentation.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)